pkg/operator/render_test.go

```diff
@@ -35,3 +35,60 @@ func TestClusterDNSIP(t *testing.T) {
 		})
 	}
 }
+
+func TestRenderAsset(t *testing.T) {
+	tests := []struct {
+		Path         string
+		RenderConfig *renderConfig
+		Error        bool
+	}{{
+		// Simple test
+		Path: "manifests/machineconfigcontroller/clusterrolebinding.yaml",
+		RenderConfig: &renderConfig{
+			TargetNamespace: "testing-namespace",
+		},
+	}, {
+		// Nested field test
+		Path: "manifests/machineconfigcontroller/deployment.yaml",
+		RenderConfig: &renderConfig{
+			TargetNamespace: "testing-namespace",
+			Images: &Images{
+				MachineConfigController: "{MCC: PLACEHOLDER}"},
```

Comment:
Looks reasonable to me but I think we could improve this by adding a member to the test struct like in this case `FindExpected: "  namespace: testing-namespace"` or so and have the test code assert that string exists in the output if `FindExpected != nil`?

Reply:
Sure, can do.